### ISSUE TITLE
Plugin is only showing three posts

### ISSUE THREAD

**Issue by soniakandah** [moduleon/grav-plugin-social-feed#9]
Hi there! 

I'm using your plugin for the first time, so apologies if I misunderstood something. But it seems that regardless of my number of connected accounts, or the limit I set in my query, only three social media posts are returned. 

`{"total":3,"limit":50,"page":1,"posts":[{"provider":"twitter","postId":"`

I believe I executed all the steps, and I am running this on a localhost using MAMP. Here is how I called the code: 

`{%
	    set postData = socialPosts()
	%}
	<ul>
	    {% for post in postData.posts %}
	    <li>{{ post.body }}</li>
	    {% endfor %}
	</ul>`

thanks for your help!

**Comment by moduleon** [comment]
Hi,

I found an issue with Instagram, apparently the endpoint used to fetch posts is no longer reachable. I will work on it.
For the others (facebook and twitter), everything works like a charm. 

Remember that there are 2 things to be defined: 
1 ) Configuration for communicating with the different apis (application keys, secrets, tokens, depending the social network you want to fetch posts from).
2) The accounts you want to follow for each social network. 

Normally, it's kind easy to define those 2 things when you fill out the configuration form of the plugin. Just make sure your configuration is complete (api key, secret, token etc are provided and corrects, username of the followed accounts too).

After that, you will have to launch the command 
```shell
bin/plugin social-feed fetch:posts
```
That will retrieve posts for each account followed in each social network fully configured. It may return some errors (invalid token, ...).

For the record, all the posts are saved in /user/data/social-feed.yaml. You should check what's in it.

**Comment by soniakandah** [comment]
Thanks for the quick response! 

I believe I'm using the correct key / secret for twitter, I'm just curious why it only fetched three posts, even though my limit is set to 50. Looks like the earliest post it fetched was from October 7th, 2018. Does the plugin only get posts for a certain period back in time? It does seem to grow to add new posts I've just made - i.e. I just posted to twitter, refreshed, and now the list had four items, with my most recent post being there.
